config/development.json

```diff
@@ -1,7 +1,9 @@
 {
   "environment": "development",
-  "logging": true,
-  "clientLogging": false,
+  "logging": {
+    "clientLogging": false,
```

Comment:
Why not just `client`? The logging suffix is redundant now that it's in a logging section.